MPSGameLift/Scripts/export_gamelift_server_package.py

```diff
@@ -0,0 +1,88 @@
+project_name = project_json_data.get('project_name')
+
+o3de_logger.info(f"Exporting AWS GameLift Server Package for {project_name}")
+
+# Parse arguments to either build code, assets, or both
+parser = argparse.ArgumentParser(
+                    prog='GameLift Server Package',
+                    description='Helps setup the project for AWS Gamelift and creates a Windows server package which can be uploaded to a GameLift.')
+
+parser.add_argument('--code', action='store_true', help='Build code')
+parser.add_argument('--assets', action='store_true', help='Build assets')
+
+args = parser.parse_args(o3de_context.args)
+
+if not args.code and not args.assets:
+    user_input = input('No build command specified. Do you want to build code, assets, or both? (c/a/b): ')
+    if user_input.lower() == 'c':
+        args.code = True
+    elif user_input.lower() == 'a':
+        args.assets = True
+    elif user_input.lower() == 'b':
+        args.code = True
+        args.assets = True
+    else:
+        print('Invalid input. Please enter c, a, or b.')
+
+# Build code
+if (args.code):
+    # Enable GameLift gems
+    o3de_logger.info(f"Enabling AWSGameLift and MPSGameLift gem")
+    if (enable_gem.enable_gem_in_project(gem_name="AWSGameLift", project_path=o3de_context.project_path) != 0):
+        quit()
+
+    if (enable_gem.enable_gem_in_project(gem_name="MPSGameLift", project_path=o3de_context.project_path) != 0):
+        quit()
+
+    # Build server launcher
+    build_folder = os.path.join(o3de_context.project_path, "build", "windows")
+    os.makedirs(build_folder, exist_ok=True)
+    o3de_logger.info(f"Building {project_name}.ServerLauncher")
+
+    if (process_command(["cmake", "-B", build_folder, "-S", o3de_context.project_path, "-G", "Visual Studio 16"])):
+        quit()
+
+    if (process_command(["cmake", "--build", build_folder, "--target", f"{project_name}.ServerLauncher", "--config", "profile", "--", "/m"]) != 0):
+        quit()
+        
+    if (process_command(["cmake", "--build", build_folder, "--target", f"AssetBundler", "--config", "profile", "--", "/m"]) != 0):
+        quit()
+
+    # Build monolithic server launcher build
+    monolithic_build_folder = os.path.join(o3de_context.project_path, "build", "windows_mono")
+    os.makedirs(monolithic_build_folder, exist_ok=True)
+    if (process_command(["cmake", "-B", monolithic_build_folder, "-S", o3de_context.project_path, "-G", "Visual Studio 16", "-DLY_MONOLITHIC_GAME=1", "-DALLOW_SETTINGS_REGISTRY_DEVELOPMENT_OVERRIDES=0"])):
+        quit()
+
+    if (process_command(["cmake", "--build", monolithic_build_folder, "--target", "MultiplayerSample.ServerLauncher", "--config", "profile", "--", "/m"])):
+        quit()
+
+# Build Assets
+if (args.assets):
+    # Process assets
+    if (process_command(["cmake", "--build", build_folder, "--target", "MultiplayerSample.Assets", "--config", "profile", "--", "/m"]) != 0):
```

Comment:
Were there any CMake targets available for the asset bundler, or will that require a separate step?

Reply:
Saw that asset bundling is marked as a future improvement. I'll still keep my question around in case other contributors can chime in here.

Reply:
Updated! Added asset bundling. This reads the game and engine's seedlists to create a full asset list, and then generates an asset bundle via the asset list.